airbyte-integrations/connectors/source-braintree-singer/.dockerignore

```diff
@@ -0,0 +1,2 @@
+build
+secrets
```

Comment:
I was mistaken about this one -- there is no need to exclude secrets since it's used in Dockerfile.test. Apologies for the confusion.